tests/queue/test_file.py

```diff
@@ -0,0 +1,83 @@
+'''
+Unit tests for `tilequeue.queue.file`.
+'''
+
+from tilequeue.queue import OutputFileQueue
+from tilequeue import tile
+from ModestMaps.Core import Coordinate
+import unittest
+import StringIO
+
+
+class TestQueue(unittest.TestCase):
+
+    def setUp(self):
+        self.test_tile_coords = [
+            (0, 0, 0),
+            (1, 2, 3),
+            (4, 5, 6),
+            (9, 3, 1),
+            (4, 7, 1)
+        ]
+        self.test_tile_objs = [
+            Coordinate(*coords) for coords in self.test_tile_coords]
+        self.tile_coords_str = '\n'.join(
+            map(tile.serialize_coord, self.test_tile_objs)) + '\n'
+        self.tiles_fp = StringIO.StringIO()
+        self.queue = OutputFileQueue(self.tiles_fp)
+
+    def test_read(self):
+        self._write_tiles_to_file()
```

Comment:
A personal preference, but I think it's clearer if _write_tiles_to_file would take in the coords explicitly instead of pulling it from `self`. I think it would be fine if each invocation just took the list of coords to write, which could just be the `self.test_tile_coords` reference.